spk/sickchill/src/service-setup.sh

```diff
@@ -1,12 +1,12 @@
 PYTHON_DIR="/var/packages/python38/target"
 PIP=${SYNOPKG_PKGDEST}/env/bin/pip3
 PATH="${SYNOPKG_PKGDEST}/bin:${SYNOPKG_PKGDEST}/env/bin:${PYTHON_DIR}/bin:${PATH}"
-HOME="${SYNOPKG_PKGDEST}/var"
-VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+HOME="${SYNOPKG_PKGVAR}"
+VIRTUALENV="${PYTHON_DIR}/bin/python3 -m venv"
```

Comment:
As `${PYTHON_DIR}/bin` is already in your path, your `VIRTUALENV` call can simply use `python3` without the need of adding the path once more.